code/__DEFINES/reactions.dm

```diff
@@ -53,16 +53,16 @@
 #define METAL_HYDROGEN_RESEARCH_MAX_AMOUNT	3000
 #define STIMULUM_RESEARCH_AMOUNT			50
 //Plasma fusion properties
-#define FUSION_ENERGY_THRESHOLD				3e9 	//Amount of energy it takes to start a fusion reaction
+#define FUSION_ENERGY_THRESHOLD				1e9 	//Amount of energy it takes to start a fusion reaction
 #define FUSION_MOLE_THRESHOLD				250 	//Mole count required (tritium/plasma) to start a fusion reaction
 #define FUSION_TRITIUM_CONVERSION_COEFFICIENT (1e-10)
 #define INSTABILITY_GAS_POWER_FACTOR 		0.003
 #define FUSION_TRITIUM_MOLES_USED  			1
 #define PLASMA_BINDING_ENERGY  				20000000
-#define TOROID_VOLUME_BREAKEVEN			1000
-#define FUSION_TEMPERATURE_THRESHOLD	    10000
+#define TOROID_VOLUME_BREAKEVEN				1000
+#define FUSION_TEMPERATURE_THRESHOLD	    9000
 #define PARTICLE_CHANCE_CONSTANT 			(-20000000)
-#define FUSION_RAD_MAX						2000
+#define FUSION_RAD_MAX						1500
 #define FUSION_RAD_COEFFICIENT				(-1000)
-#define FUSION_INSTABILITY_ENDOTHERMALITY   2
+#define FUSION_INSTABILITY_ENDOTHERMALITY   3
```

Comment:
Standardizing toroidal size would make 3:1 ratio fusion mixes standardized. Bumping this up to 3 makes it too hugbox in my opinion, not to mention you're basically allocating roughly 72.66% of the entire state space to exothermic reactions. Paired with the removal of oxygen from byproduct emissions, this makes fusion far too easy.